inbox/react/get-started.mdx

```diff
@@ -48,26 +48,6 @@ Possible reasons for the Inbox not loading properly:
     - Invalid subscriberId
     - Invalid applicationIdentifier
     - Invalid subscriberHash (in case of active HMAC encryption)
-    - Invalid backendUrl (in case of self-hosted)
-    - Invalid socketUrl (in case of self-hosted)
-
-  </Accordion>
-
-  <Accordion title="There is a render error due to the Inbox in React.">
-
-The Inbox should be wrapped in `<Inbox>`
-
-  </Accordion>
-
-  <Accordion title="What is the difference between a redirect URL and CTA?">
-
-The redirect URL is for the entire notification item, When the user clicks the notification, the user will be routed to that url. CTA are two call-to-action buttons. `onNotificationClick` function prop is used for redirecting url and the `onActionClick` function prop is used for CTA.
-
-  </Accordion>
-
-  <Accordion title="Can bell icon in Novu dashboard be used for in-app channel testing?">
-
-Bell icon and Inbox in Novu dashboard is only for receiving system related notifications like user you invited has joined, password reset request sent. It can not to used to test Inbox. Consider using our [codesandbox example](https://codesandbox.io/s/kind-nova-xv6s9g?fontsize=14&hidenavigation=1&theme=dark) instead.
```

Comment:
Not relevant for Inbox

Reply:
The `backendUrl` and `socketUrl`  could be specified on the Inbox component.
The section about redirect URL, `onNotificationClick`, and action handlers `onPrimaryActionClick, onSecondaryActionClick` should be updated.

Reply:
Adding back `socketUrl` and `backendUrl` info. I think you added other info in recent PR.